fix(esl-media): make video unfocusable if `esl-media[role=presentation]` (#2829)
exadel-inc/esl: src/modules/esl-media/README.md

 - `start-time` - attribute that allows a user to start viewing a video from a specific time offset. The value is simple time in seconds. By default, it is undefined which means to start from the beginning.
*(note: that feature doesn't work for Abstract Iframe provider, also doesn't work for HTMLAudio and HTMLVideo providers in case when Web-server when hosted resource doesn't support ['Accept-Ranges' HTTP response marker](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Ranges))*

>  - `focusable` (boolean) - marker that allows the video to be focused by keyboard navigation. By default, the video is not focusable controls are disabled.

Reviewer: ```suggestion
 - `focusable` (boolean) - marker that allows the video to be focused by keyboard navigation. By default, the video is focusable if `controls` are enabled.
```
